src/f32/vec3_mask.rs

```diff
@@ -31,6 +34,33 @@ impl Default for Vec3Mask {
     }
 }
 
+#[cfg(vec3sse2)]
+impl PartialEq for Vec3Mask {
+    fn eq(&self, other: &Self) -> bool {
+        let self_arr = unsafe { &*(self as *const Vec3Mask as *const [f32; 3]) };
+        let other_arr = unsafe { &*(other as *const Vec3Mask as *const [f32; 3]) };
```

Comment:
It might be worth adding `impl From<Vec3Mask> for [u32; 3]` for `Vec3Mask` to make this a bit easier. You cmight be able to cast straight to `*const [u32; 3]`. Otherwise there are also intrinsic methods to cast `_m128` to `m128i` (signed int) which is a no-op, then cast to u32 from that.